src/Cli/dotnet/ToolPackage/ToolPackageDownloader.cs

```diff
@@ -178,6 +189,51 @@ internal class ToolPackageDownloader : IToolPackageDownloader
                 });
         }
 
+        private static void IsRuntimeConfigCompatible(
+            ToolPackageInstance toolPackageInstance,
+            PackageId packageId
+            )
+        {
+            var executableFilePath = toolPackageInstance.Commands[0].Executable;
+            var runtimeConfigFilePath = Path.ChangeExtension(executableFilePath.ToString(), ".runtimeconfig.json");
+
+            // Update the runtimeconfig.json file
+            // TBD: read tfm from runtimeconfig.json
+            string tfmValue = "";
+            if (File.Exists(runtimeConfigFilePath))
+            { 
+                JsonElement rootElement = JsonDocument.Parse(File.ReadAllText(runtimeConfigFilePath)).RootElement;
+                
+                if (rootElement.TryGetProperty("runtimeOptions", out JsonElement runtimeOptionsElement) &&
+                runtimeOptionsElement.TryGetProperty("tfm", out JsonElement tfmElement))
+                {
+                    tfmValue = tfmElement.GetString();
+                }
+                var result = NETCoreSdkResolverNativeWrapper.InitializeForRuntimeConfig(runtimeConfigFilePath);
+                RuntimeConfigDetectionMessage(result, packageId, tfmValue);
+            }
+        }
+
+        private static void RuntimeConfigDetectionMessage(int result, PackageId packageId, string tfmValue = "")
+        {
+            switch(result)
+            {
+                // RuntimeConfigDetectionResult::Success
+                case 0:
+                case 1:
+                case 2:
+                    break;
+
+                // RuntimeConfigDetectionResult incompatible
+                // TBD: if include -g in sample command; which version should be included in the install; --force?
```

Comment:
Tests should be included for `--force` and `IsRuntimeConfigCompatible` functions

Reply:
It looks like `--force` was added but maybe not `IsRuntimeConfigCompatible`.